webapp/i18n/fr.json

```diff
@@ -968,6 +968,18 @@
   "analytics.team.title": "Statistiques d'équipe pour {team}",
   "analytics.team.totalPosts": "Nombre de messages",
   "analytics.team.totalUsers": "Nombre d'utilisateurs",
+  "api.channel.add_member.added": "{addedUsername} a été ajouté au canal par {username}",
```

Comment:
I just copied the French ones from the server so I could test out the client-side rendering.

@enahum Is it okay if these are included in this PR or should I only keep the English ones?